setup.py

```diff
@@ -1,28 +1,41 @@
+#!/usr/bin/env python
 # -*- coding: utf-8 -*-
+from os.path import dirname, join
+
 from setuptools import setup, find_packages
 
+import melissa
+
+description = 'A lovely virtual assistant for OS X, Windows and Linux systems.'
 try:
     long_description = open("README.rst").read()
 except IOError:
-    long_description = ""
+    long_description = description
 
 setup(
     name="melissa",
     version="0.1.0",
-    description="melissa description",
+    description=description,
```

Comment:
that only apply to variable assignment. function argument not require that.

also pytest-flake8 isn't reporting anything, so it is ok